boards/arm/v2m_beetle/doc/v2m_beetle.rst

```diff
@@ -237,7 +237,7 @@ the :ref:`hello_world` application.
 .. zephyr-app-commands::
    :zephyr-app: samples/hello_world
    :board: v2m_beetle
-   :goals: build flash
+   :goals: build
```

Comment:
I don't see any instructions here that tell the reader how to flash?

Reply:
They're in the very next paragraph.

The board does not support 'make flash'. The cmake conversion added that by mistake. That is the error which this patch is correcting.

Reply:
Here is my commit which made the mistake of adding "flash", which was merged during the cmake conversion. Note "make flash" is not present in the original text.

```
commit f0c95919b06fa64c356cb1713b6e103724e922f3
Author: Marti Bolivar <marti@opensourcefoundries.com>
Date:   Tue Nov 7 22:44:42 2017 -0500

    doc: boards: arm: convert make to cmake
    
    Unify documentation formatting and use zephyr-app-commands where
    applicable.
    
    Signed-off-by: Marti Bolivar <marti@opensourcefoundries.com>

diff --git a/boards/arm/v2m_beetle/doc/v2m_beetle.rst b/boards/arm/v2m_beetle/doc/v2m_beetle.rst
index beba0b718..cf6dcdc08 100644
--- a/boards/arm/v2m_beetle/doc/v2m_beetle.rst
+++ b/boards/arm/v2m_beetle/doc/v2m_beetle.rst
@@ -231,20 +231,13 @@ to `CMSIS-DAP Website`_.
 Flashing an application to V2M Beetle
 -------------------------------------
 
-The sample application :ref:`hello_world` is being used in this tutorial:
+You can build applications in the usual way. Here is an example for
+the :ref:`hello_world` application.
 
-.. code-block:: console
-
-   $ZEPHYR_BASE/samples/hello_world
-
-To build the Zephyr kernel and application, enter:
-
-.. code-block:: console
-
-   $ cd $ZEPHYR_BASE
-   $ . zephyr-env.sh
-   $ cd $ZEPHYR_BASE/samples/hello_world/
-   $ make BOARD=v2m_beetle
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: v2m_beetle
+   :goals: build flash
 
 Connect the V2M Beetle to your host computer using the USB port and you should
 see a USB connection which exposes a Mass Storage (MBED) and a USB Serial Port.
```

Reply:
Just for reference, this is the link to our discussion in the PR for 1.11: https://github.com/zephyrproject-rtos/zephyr/pull/5232#discussion_r154474902

Reply:
Doh!